### DISCUSSION TITLE
Understanding the vertex information units

### DISCUSSION THREAD

**Discussion by zohaibjaved2** [assimp/assimp#4031] [Q&A] [answered]
Hello, 

I am trying to load a 1 cm cube model exported from Maya. I was able to successfully load it in unreal through following code:

`FString libPath = FPaths::ConvertRelativePathToFull(FPaths::Combine(*FPaths::ProjectDir(), TEXT("Source/ThirdParty/bin/Windows")));
	FPlatformProcess::PushDllDirectory(*libPath);
	
	UE_LOG(LogMesh, Log, TEXT("Assimp version : %d,%d,%d"), aiGetVersionMajor(),aiGetVersionMinor(),aiGetVersionRevision());

	Assimp::Importer importer;
	importer.SetPropertyInteger(AI_CONFIG_PP_SLM_VERTEX_LIMIT, TNumericLimits<int>::Max());
	importer.SetPropertyInteger(AI_CONFIG_PP_SLM_TRIANGLE_LIMIT, TNumericLimits<int>::Max());
	importer.SetPropertyBool(AI_CONFIG_PP_FD_REMOVE, true);
	importer.SetPropertyBool(AI_CONFIG_IMPORT_FBX_READ_ANIMATIONS, false);
	importer.SetPropertyBool(AI_CONFIG_IMPORT_FBX_READ_CAMERAS, false);
	importer.SetPropertyBool(AI_CONFIG_IMPORT_FBX_READ_TEXTURES, false);
	importer.SetPropertyInteger(AI_CONFIG_PP_SBP_REMOVE, aiPrimitiveType_LINE | aiPrimitiveType_POINT);
	importer.SetPropertyBool(AI_CONFIG_PP_PTV_KEEP_HIERARCHY, true);
	
	int postProcessSteps = aiProcess_CalcTangentSpace |
							aiProcess_JoinIdenticalVertices |
							aiProcess_Triangulate |
							aiProcess_GenUVCoords |
							aiProcess_SortByPType |
							aiProcess_FindDegenerates |
							aiProcess_FindInvalidData |
							aiProcess_FindInstances |
							aiProcess_OptimizeMeshes |
							aiProcess_FlipWindingOrder |
							aiProcess_GlobalScale |
							aiProcess_MakeLeftHanded | 
							aiProcess_PreTransformVertices;

	std::string fileName_str(TCHAR_TO_UTF8(*_filename));
	
	_assimpScene = importer.ReadFile(fileName_str, postProcessSteps);` 

When I try to see the vertex information I get very small value like 0.00499999989. It seems like by default we get units in meters. Is this correct assumption? 

Also I want to do the calculations in centimeters. Is there a way to convert it using some importer settings?

**Comment by kimkulling** [comment, marked as answer]
At this moment Assimp does not support any global unit-setting. Some importers will provide this information in their metadata. 
I will open a new PR with this feature. We got this question quite often. So we need a solution for that.
